applications/launchpad/gui-vue/package.json

```diff
@@ -10,25 +10,15 @@
   },
   "dependencies": {
     "@fortawesome/fontawesome-svg-core": "^1.2.36",
-    "@fortawesome/free-brands-svg-icons": "^5.15.4",
-    "@fortawesome/free-regular-svg-icons": "^5.15.4",
     "@fortawesome/free-solid-svg-icons": "^5.15.4",
     "@fortawesome/vue-fontawesome": "^3.0.0-5",
     "@oruga-ui/oruga-next": "^0.4.8",
     "@tauri-apps/api": "^1.0.0-beta.8",
     "CBuffer": "^2.2.0",
-    "core-js": "^3.6.5",
     "vue": "^3.0.0",
     "vuex": "^4.0.2"
   },
   "devDependencies": {
-    "@tauri-apps/cli": "^1.0.0-beta.10",
```

Comment:
I'm really only concerned with this one - it looks like `npm run tauri` does actually use the tauri cli. Can you confirm?

Reply:
Yeah., I've added it back.